test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -36,10 +38,12 @@ class PostBindingVerifier(private val response: IdpPostResponseDecorator) {
      * Verify the response for a post binding
      */
     fun verify() {
+        Log.debug("Starting the Binding Verifications for the POST Binding.")
         decodeAndVerify()
         verifySsoPost()
         verifyPostRelayState()
         verifyPostDestination()
+        Log.debug("Completed the Binding Verifications for the POST Binding.")
```

Comment:
✏️ I'm not sure I see a lot of value in a ton of these "trace" type logs. With the exception of the tests. I think it's good for us to have that for the tests.

Reply:
Agreed.

Reply:
Do you mean keep just the starting post/redirect test logs or move these to the tests?

Reply:
No I mean get rid of this line and the similar ones but keep the ones you already have in the tests.